This file is part of mingw-cross-env.
See doc/index.html for further information.

Patch taken from
https://sourceforge.net/tracker/?func=detail&aid=3288762&group_id=52781&atid=468023

From d49ac167169e2db531e1be5a48575186c8bc8799 Mon Sep 17 00:00:00 2001
From: mingw-cross-env
Date: Mon, 18 Apr 2011 09:20:27 +0200
Subject: [PATCH] use ws2spi instead of wspiapi


diff --git a/gsoap/VisualStudio2005/wsdl2h/wsdl2h/stdsoap2.h b/gsoap/VisualStudio2005/wsdl2h/wsdl2h/stdsoap2.h
index 238b7f1..ffccf80 100644
--- a/gsoap/VisualStudio2005/wsdl2h/wsdl2h/stdsoap2.h
+++ b/gsoap/VisualStudio2005/wsdl2h/wsdl2h/stdsoap2.h
@@ -693,7 +693,7 @@ A commercial use license is available from Genivia, Inc., contact@genivia.com
 # endif
 #  include <winsock2.h> /* Visual Studio 2005 users: you must install the Platform SDK (R2) */
 #  include <ws2tcpip.h>
-#  include <wspiapi.h>
+#  include <ws2spi.h>
 # ifdef WITH_IPV6
 #  define SOAP_GAI_STRERROR gai_strerrorA
 # endif
diff --git a/gsoap/samples/calc_vs2005/calc_vs2005/stdsoap2.h b/gsoap/samples/calc_vs2005/calc_vs2005/stdsoap2.h
index 238b7f1..ffccf80 100644
--- a/gsoap/samples/calc_vs2005/calc_vs2005/stdsoap2.h
+++ b/gsoap/samples/calc_vs2005/calc_vs2005/stdsoap2.h
@@ -693,7 +693,7 @@ A commercial use license is available from Genivia, Inc., contact@genivia.com
 # endif
 #  include <winsock2.h> /* Visual Studio 2005 users: you must install the Platform SDK (R2) */
 #  include <ws2tcpip.h>
-#  include <wspiapi.h>
+#  include <ws2spi.h>
 # ifdef WITH_IPV6
 #  define SOAP_GAI_STRERROR gai_strerrorA
 # endif
diff --git a/gsoap/stdsoap2.h b/gsoap/stdsoap2.h
index 238b7f1..ffccf80 100644
--- a/gsoap/stdsoap2.h
+++ b/gsoap/stdsoap2.h
@@ -693,7 +693,7 @@ A commercial use license is available from Genivia, Inc., contact@genivia.com
 # endif
 #  include <winsock2.h> /* Visual Studio 2005 users: you must install the Platform SDK (R2) */
 #  include <ws2tcpip.h>
-#  include <wspiapi.h>
+#  include <ws2spi.h>
 # ifdef WITH_IPV6
 #  define SOAP_GAI_STRERROR gai_strerrorA
 # endif
-- 
1.7.4.1

